its/core-it-suite/src/test/java/org/apache/maven/it/MavenITmng6401ProxyPortInterpolationTest.java

```diff
@@ -28,10 +28,6 @@
 
 class MavenITmng6401ProxyPortInterpolationTest extends AbstractMavenIntegrationTestCase {
 
-    private Proxy proxy;
```

Comment:
These might be used in a weird way by reflection in the integration tests. I'm not sure in this case, but Maven does have a lot of private things that look unused but really are used like that.

Reply:
yes then its nice to have good TC to challenge or suppress.